src/aggregator/aggregator/follower_flush_mgr.go

```diff
@@ -241,6 +241,20 @@ func (mgr *followerFlushManager) CanLead() bool {
 		mgr.metrics.notCampaigning.Inc(1)
```

Comment:
@justinjc for my understanding: we're not campaigning if we're not cutover right?